cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,51 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
-  });
-
-  it('', () => {
-    
+  describe('User data', () => {
+    const user = {
+      userName: 'AnnaTest',
+      password: 'AnnaTest!123'
+    }
+    it('login', () => {
```

Comment:
please, write the correct test name. it is not very clear what it checks (in the whole file)